examples/simple_regional_beta/main.tf

```diff
@@ -49,6 +49,7 @@ module "gke" {
   database_encryption         = var.database_encryption
   enable_binary_authorization = var.enable_binary_authorization
   enable_pod_security_policy  = var.enable_pod_security_policy
+  enable_identity_service     = var.enable_identity_service
```

Comment:
You should just hardcode this here in the example. Wiring variables through is an anti-pattern we're trying to stop using.